### PR TITLE
[change] Updated the serializers imports #287

### LINKED ISSUE #287: Update imports of geo API serializers from openwisp-controller
Update the import of `serializers` according to https://github.com/openwisp/openwisp-controller/pull/416 change.

https://github.com/openwisp/openwisp-monitoring/blob/fd6b200a96a601e8508af52c2320e1f8d1dbd91d/openwisp_monitoring/device/api/views.py#L23

### PR DESCRIPTION
**Pull request by ManishShah120**
closes #287

Checks:

- [x] I have manually tested the proposed changes
- [ ] I have written new test cases to avoid regressions (if necessary)
- [ ] I have updated the documentation (e.g. README.rst)

### CODE REVIEW

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
@ManishShah120 also here there are some serializers mixed with views, can you fix it please?
Also point the requirements.txt to the branch where you did the change so the build should pass.

**Comment by ManishShah120** [comment]
Sure will do it.

**Comment by ManishShah120** [comment]
Done @nemesisdesign

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # TODO: put back the standard pypi release when
 # next point version of openwisp-controller is released
-openwisp-controller @ https://github.com/openwisp/openwisp-controller/tarball/master
+openwisp-controller @ https://github.com/ManishShah120/openwisp-controller/tarball/issues/414-separate-serializer-classes
```
Comment (nemesisdesign) [inline requirements.txt:3]: I just merged https://github.com/openwisp/openwisp-controller/pull/416, now we can switch this to master

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Thanks @ManishShah120, please see my comment below and then we can merge!

**Review by nemesisdesign — APPROVED** [review]
I resolved the pending issue since I could do it easily, thanks @ManishShah120 :+1:

**Pull request merged**